src/pages/gen2/build-a-backend/data/custom-business-logic/index.mdx

```diff
@@ -4,24 +4,17 @@ export const meta = {
     'Customize your business logic for queries, mutations, and subscriptions.'
 };
 
-export function getStaticProps(context) {
+export function getStaticProps() {
   return {
     props: {
       meta
     }
   };
-
 }
 
-<Callout info>
-
-**Coming soon:** The custom business logic experience in Amplify (Gen 2) is still under development. Expect the API surface to change dramatically during the Gen-2 preview period. If you have any feedback on the desired developer experience, please [file a GitHub issue](https://github.com/aws-amplify/amplify-backend).
-
-</Callout>
-
-The default GraphQL API provides a solid foundation for querying, mutating, and fetching data. Even so, you may need additional customization to tailor it to your app's specific requirements around data handling, response formatting, and more.
+The `a.model()` data model provides a solid foundation for querying, mutating, and fetching data. However, you may need additional customizations to meet specific requirements around custom API requests, response formatting, and or fetching from external data sources.
 
-In the following sections, we walk through the three steps to create a custom mutation or query:
+In the following sections, we walk through the three steps to create a custom query or query:
```

Comment:
Did you mean to remove "mutation" altogether, or was this an accidental duplication?